thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/tasks/FunctionCreationOnboardingTask.java

```diff
@@ -298,6 +322,21 @@ public void run() {
     executionContext.setExecutionResult(ALERT_CONFIG, alertConfig);
   }
 
+  private AnomalyFunctionDTO getDefaultFunctionSpec(boolean isLegacy, TimeGranularity dataGranularity) {
+    AnomalyFunctionDTO anomalyFunctionSpec = new AnomalyFunctionDTO();
+    if (isLegacy) {
+      anomalyFunctionSpec = getDefaultFunctionSpecByTimeGranularity(dataGranularity);
+    } else {
+      anomalyFunctionSpec.setType("SPLINE_REGRESSION_WRAPPER");
```

Comment:
does this work when time granularity is minutes or hours?

Reply:
This endpoint works only for daily metrics until we have the new algorithms supporting MIN and HOUR granularity.

Reply:
ok I'll throw an Exception on non-daily granularity then.